docs/adding_packages/dependencies.md

```diff
@@ -192,6 +196,10 @@ version range only when a requirement for a newer version is needed.
 * qt5: `[~5.15]`, if your library depends on qt5, only the 5.15 minor version is allowed
 * qt6: `[>=6.x <7]`, where 6.x is the lower bound of your needed qt6 version
 * c-ares: `[>=1.27 <2]`
+* zstd: `[^1.5]` it's equivalent to `[>=1.5 <1.6]`
```

Comment:
AFAIK, `[^1.5]` it's equivalent to `[>=1.5 <2]`

Reply:
You are right. https://docs.conan.io/1/versioning/version_ranges.html#version-ranges I'll open a PR fixing it. Thank you!